tools/myst/common.c

```diff
@@ -40,16 +42,17 @@ int myst_expand_size_string_to_ulong(const char* size_string, size_t* size)
 
 bool myst_merge_mount_mapping_and_config(
     myst_mounts_config_t* mounts,
-    myst_mount_mapping_t* mount_mapping)
+    myst_args_t* mount_mapping)
 {
     bool ret = false;
 
-    for (size_t i = 0; i < mount_mapping->mounts_count; i++)
+    for (size_t i = 0; i < mount_mapping->size; i++)
     {
         char* source;
         char* target;
         bool found = false;
-        source = strtok(mount_mapping->mounts[i], "=");
+
+        source = strtok((char*)(mount_mapping->data[i]), "=");
```

Comment:
It probably doesn't matter because this is a single threaded program, but I would probably use the thread-safe form: ``strtok_r``.

Reply:
fixed